airflow_clickhouse_plugin/hooks/clickhouse_hook.py

```diff
@@ -46,7 +48,7 @@ def get_first(self, sql: str, parameters: Optional[dict] = None) -> Optional[Tup
                 return None
 
     def get_pandas_df(self, sql: str):
-        import pandas as pd
+        import pandas as pd # pylint: disable=import-outside-toplevel
```

Comment:
Please do not include your personal pylint noqas into the repo. Everyone uses their own code quality tool.